src/server/api/routes/users/v2/email.js

```diff
@@ -73,20 +74,22 @@ router.post('/email', authenticateV2Login, async (req, res, next) => {
       qsPayload.payment_id = paymentId;
       qsPayload.claiming_token = claimingToken;
     }
-    const verificationURL = `${EXTERNAL_URL}/settings/email/verify/${token}?${querystring.stringify(
-      qsPayload
-    )}`;
-    const title = 'Verify Your Email';
-    const { subject, body } = getBasicV2Template({
-      subject: title,
-      title,
-      content: `<p>Please click the link to verify your email:</p><p>${verificationURL}</p>`,
-    });
-    await sendEmail({
-      email,
-      subject,
-      html: body,
-    });
+    if (verify !== '0') {
```

Comment:
Maybe add a comment here?

Reply:
about the meaning of `verify`, or when do we set `verify === '0'` ?
Maybe we should rename `verify` if its meaning is unclear

Reply:
How about this
```suggestion
    const hasVerifiedInAuthcore = verify !== '0';
    if (hasVerifiedInAuthcore) {
```

Reply:
Renamed var and added authcore description